src/libultra_code/osReadMempak.c

```diff
@@ -0,0 +1,57 @@
+#include <ultra64.h>
+#include <global.h>
+
+#include <ultra64/controller.h>
+
+s32 osReadMempak(OSMesgQueue* ctrlrqueue, s32 ctrlridx, u16 addr, PIF_mempak_data_t* data) {
+    s32 ret;
+    s32 i;
+    u8* bufptr;
+    s32 read_try_count = 2;
+
+    __osSiGetAccess();
+    do {
+        if ((_osCont_lastPollType != 2) || (ctrlridx != D_80134D20)) {
+            bufptr = &pifMempakBuf.bytes[0];
+            _osCont_lastPollType = (u8)2U;
+            D_80134D20 = ctrlridx;
+            // clang-format off
+            // whitespace memes - only matches on same line
```

Comment:
This should need a special comment to prevent clang-tidy from adding braces here. I also don't think you need to explain the need for them to be on the same line to match, since that should be pretty obvious from the format off comments.
```suggestion
            // NOLINTNEXTLINE
```

Reply:
I added your `NOLINTNEXTLINE` suggestion, but with `clang-format off`, format.sh didn't mess up that code, so I'm not sure why it's needed. Also, the braces don't cause it to not match--moving it to the next line causes it to not match.

Reply:
I just tested it, and format.sh definitely runs clang-tidy on this line and messes it up if there is no `NOLINTNEXTLINE` comment. And yeah the braces are fine on their own, but since clang-format is off in that section, clang-tidy will add braces in bad places (clang-tidy relies on clang-format under the hood to format braces properly after adding them).

Reply:
Welp, there must be some machine configuration-specific options going on here, because I ran format.sh without NOLINTNEXTLINE and it didn't change it. But I've already already added it anyway.